packages/mdl-drawer/README.md

```diff
@@ -0,0 +1,284 @@
+# MDL Drawer
+
+The MDL Drawer component is a spec-aligned drawer component adhering to the
+[Material Design navigation drawer pattern](https://material.google.com/patterns/navigation-drawer.html).
+It implements permanent and temporary drawers. Permanent drawers are CSS-only and require no JavaScript, whereas
+temporary drawers require JavaScript to function, in order to respond to user interaction.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Permanent drawer usage
+
+A permanent drawer is always open, sitting to the side of the content. It is appropriate for any display size larger
+than mobile.
+
+> TODO(sgomes): Give advice on how to hide permanent drawer in mobile.
+
+```css
+body {
```

Comment:
We may want to also mention and/or show an example of using a top-level wrapper element, which developers may feel more comfortable changing the overall display of than the body. 

Also, is flex _required_ for the drawer? Or just for the example with the toolbar/content layout? If flex isn't required then perhaps we should think about paring down the examples such that it omits the toolbar and content and just shows an "open drawer" button.

We could then create a higher-level layout module, or just provide templates showing how to implement our common patterns. WDYT?